server/routerlicious/packages/services-client/src/gitManager.ts

```diff
@@ -257,7 +257,7 @@ export class GitManager implements IGitManager {
                         entryAsBlob.contents = this.translateSymlink(entryAsBlob.contents, depth);
                     }
 
-                    const blobP = this.createBlob(entryAsBlob.contents, entryAsBlob.encoding);
+                    const blobP = this.createBlob(entryAsBlob.contents, entryAsBlob.encoding as BufferEncoding);
```

Comment:
Ah, good...   I was hoping there would be an example where we need to publish an updated 'pre-release' of protocol-definitions.  Once that lands, you'll be able to remove this cast.